src/kibana/components/vislib/visualizations/line_chart.js

```diff
@@ -332,15 +320,17 @@ define(function (require) {
           self.addCircleEvents(circles, svg);
           self.createEndZones(svg);
 
-          var line = svg
-          .append('line')
-          .attr('class', 'base-line')
-          .attr('x1', startLineX)
-          .attr('y1', yScale(0))
-          .attr('x2', width)
-          .attr('y2', yScale(0))
-          .style('stroke', '#ddd')
-          .style('stroke-width', lineStrokeWidth);
+          var scale = (scaleType === 'log') ? yScale(1) : yScale(0);
```

Comment:
I am not sure `yScale(1)` is the best choice here. In cases where the user selects a log scale and the sets the y axis min value to something greater than 1, you still won't see the zero line. Usually, it is more common to take the height of the svg container.